drake/systems/plants/RigidBodySystem.cpp

```diff
@@ -367,8 +361,53 @@ RigidBodySpringDamper::RigidBodySpringDamper(RigidBodySystem* sys,
   tree->addFrame(frameB);
 }
 
+RigidBodyAccelerometer::RigidBodyAccelerometer(RigidBodySystem const& sys,
+                                               const std::string& name,
+                                               const std::shared_ptr<RigidBodyFrame> frame)
+  : RigidBodySensor(sys, name), frame(frame) {  }
+
+
+Eigen::VectorXd RigidBodyAccelerometer::output(const double &t,
+                                               const KinematicsCache<double> &rigid_body_state,
+                                               const RigidBodySystem::InputVector<double>& u) const
+{
+  auto const& q = rigid_body_state.getQ();
+  auto const& v = rigid_body_state.getV();
+  VectorXd x(q.size() + v.size());
+  x << q, v;
```

Comment:
we should have a rigid_body_state.getX() so you don't have to do this everywhere.  (i didn't actually know we had getQ and getV; there are probably places where I should have used those!)